api_tests/src/actors/defaults.ts

```diff
@@ -0,0 +1,173 @@
+import { HarnessGlobal } from "../utils";
+import { Peer } from "comit-sdk";
+import {
+    HalightLightningBitcoinHanEthereumEtherRequestBody,
+    HalightLightningBitcoinHerc20EthereumErc20RequestBody,
+    HanEthereumEtherHalightLightningBitcoinRequestBody,
+    Herc20EthereumErc20HalightLightningBitcoinRequestBody,
+} from "comit-sdk/dist/src/cnd/cnd";
+import { Asset, AssetKind } from "../asset";
+import { Ledger, LedgerKind } from "../ledgers/ledger";
+import { parseEther } from "ethers/utils";
+
+declare var global: HarnessGlobal;
+
+export function defaultHanEthereumEtherHalightLightningBitcoin(
+    lndPubkey: string,
+    peer: Peer
+): HanEthereumEtherHalightLightningBitcoinRequestBody {
+    return {
+        alpha_amount: "5000000000000000000",
+        beta_amount: "10000000",
+        alpha_network: "17",
+        beta_network: "regtest",
+        alpha_identity: "0x00a329c0648769a73afac7f9381e08fb43dbea72",
+        beta_identity: lndPubkey,
+        role: "Alice",
+        peer,
+        ...defaultHalightHanHerc20Expiries(),
+    };
+}
+
+export function defaultHerc20EthereumErc20HalightLightningBitcoin(
+    lndPubkey: string,
+    peer: Peer
+): Herc20EthereumErc20HalightLightningBitcoinRequestBody {
+    return {
+        alpha_amount: "9000000000000000000",
+        alpha_contract_address: "0xB97048628DB6B661D4C2aA833e95Dbe1A905B280",
+        beta_amount: "10000000",
+        alpha_network: "17",
+        beta_network: "regtest",
+        alpha_identity: "0x00a329c0648769a73afac7f9381e08fb43dbea72",
+        beta_identity: lndPubkey,
+        role: "Alice",
+        peer,
+        ...defaultHalightHanHerc20Expiries(),
+    };
+}
+
+export function defaultHalightLightningBitcoinHanEthereumEther(
+    lndPubkey: string,
+    peer: Peer
+): HalightLightningBitcoinHanEthereumEtherRequestBody {
+    return {
+        alpha_amount: "10000000",
+        beta_amount: "5000000000000000000",
+        alpha_network: "regtest",
+        beta_network: "17",
```

Comment:
Are we happy with that or do we want to update the current design?

Reply:
Let's roll with this for now.